app/assets/javascripts/misc/recovery-page-controller.js

```diff
@@ -42,7 +42,7 @@ function handleSubmit(event) {
   const invalidMatches = inputs.reduce(function(accumulator, input, index) {
     const value = input.value;
 
-    if (value === recoveryWords[index].innerHTML.replace(/\s+/, '')) {
+    if (value.toUpperCase() === recoveryWords[index].innerHTML.replace(/\s+/, '').toUpperCase()) {
```

Comment:
I am reluctant to start trying to re-implement and maintain the crockford rules in JS. Besides, isn't the point of the crockford generation is to *avoid* creating codes that contain those problematic characters? So there won't be codes with the letters `ILOU` in them?